### ISSUE TITLE
Turn off code completion for certain directories

### ISSUE THREAD

**Issue by brychta-v** [juzna/intellij-neon#46]
I use neon files to store translations and it's really annoing for me to edit them, because every translation is provided with autocompletion of my classes.

It would be nice to have an option to turn off the autocompletion for certain directories, e.g. when the directory is marked as excluded.

**Comment by JanTvrdik** [comment]
Or maybe disable code completion unless explicitly requested (ctrl + space)?

**Comment by hrach** [comment]
This is something what should be done by splitting neon as two plugins - di configuration file & raw neon file, isn't it?

**Comment by JanTvrdik** [comment]
@hrach Yes, that was the originally plan when this repo for was created =)
